MyOnboarding/test/MyOnboardingApp.Api.Tests/App_Start/DependenciesConfigTests.cs

```diff
@@ -33,8 +45,12 @@ public void UnityContainer_AfterDependencyRegistration_ContainsAllContracts()
             var container = MockUnityContainer(actualTypes);
 
             DependenciesConfig.RegisterAllDependencies(container);
```

Comment:
Hm, any specific reason not to call this method as an extension?
```suggestion
            container.RegisterAllDependencies();
```